modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/FileEntryPicker.js

```diff
@@ -21,20 +21,33 @@ import React, {useEffect, useState} from 'react';
 
 const FileNamePicker = ({
 	maxFileSize: initialMaxFileSize,
+	maxMimeTypeSize,
 	namespace,
 	validExtensions,
 }) => {
-	const maxFileSize = Number(initialMaxFileSize);
+	let maxFileSize = Number(initialMaxFileSize);
 	const inputId = `${namespace}file`;
 	const [inputValue, setInputValue] = useState('');
 	const [fileName, setFileName] = useState('');
 	const [maxFileSizeError, setMaxFileSizeError] = useState(false);
-
+	const [maxSize, setMaxSizeValue] = useState(maxFileSize);
 	useEffect(() => {
 		setFileName(inputValue ? inputValue.replace(/^.*[\\]/, '') : '');
 	}, [inputValue]);
 
 	const onInputChange = ({target}) => {
+		const jsonObj = JSON.parse(maxMimeTypeSize);
+		const type = target.files[0].type;
+		const maxMimeSize = Number(jsonObj[type]);
```

Comment:
@thienquach87 
This will make sure the default value is 0 ? or undefined?

Reply:
we don't need check it in reactjs or js